docs/design/management_plane.md

```diff
@@ -336,15 +352,82 @@ For example:
+5. The Divider Operator removes all Dividers in the VPC from its cache.
+6. The Divider Operator calls back to the VPC Operator after all Dividers have been deleted.
+7. The VPC Operator Deletes the VPC Object.
+8. The VPC Operator remoes the VPC object from its cache
+
 ### Network Object Delete Workflow
-![Network](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/phudtran/mizar/dev-k8s/Documentation/design/figures/delete_network_workflow.puml)
+![Network](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/futurewei-cloud/mizar/dev-next/docs/design/puml/delete_network_workflow.puml)
+
+1. The Network Operator sees a Network delete request
+2. The Network Operator will not delete the Network unless all of the specified Network's children Endpoints have been deleted.
+3. The Network Operator calls back to the Divider Operator after all Endpoints have been deleted.
+4. The Divider Operator deletes all Divider information from all Bouncers.
+5. The Divide Operator deletes all Bouncer information from the Dividers
+6. The Divider calls back to the Bouncer Operator after all Bouncers have been deprovisioned.
+7. The Bouncer Operator deletes all Bouncer Objects in the Network.
+8. The Bouncer Operator removes all Bouncer Objects in the Network from its cache.
+9. The Bouncer Operator calls back to the Network Operator that all Bouncers in the Network have been deleted.
+10. The Network Operator deletes the Network Object.
+11. The Network Operator removes the Network object from its cache.
+
 ### Endpoint Object Delete Workflow
-![Endpoint](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/phudtran/mizar/dev-k8s/Documentation/design/figures/delete_endpoint_workflow.puml)
+![Endpoint](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/futurewei-cloud/mizar/dev-next/docs/design/puml/delete_endpoint_workflow.puml)
+
+1. The CNI Services sees the Endpoint Delete request.
+2. The CNI service deletes the Endpoint Object.
+3. The Network Operator sees the Endpoint Delete request.
+4. The Network operator deallocates the Endpoints information (IP, MAC, etc).
+5. The Network Operator calls back to the Bouncer Operator after the Endpoint has been deallocated.
+6. The Bouncer Operator deletes the Endpoint information from all Bouncers.
+7. The Bouncer Operator deletes all Bouncer information from the Endpoint.
+8. The Bouncer Operator calls back to the Endpoint Operator.
+9. The Endpoint Operator deletes the Endpoint Object from its cache.
+
+### Scaled Endpoint Object Delete Workflow
+![Scaled_Endpoint](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/futurewei-cloud/mizar/dev-next/docs/design/puml/delete_scaled_endpoint_workflow.puml)
+
+1. The CNI Services sees the Scaled Endpoint Delete request.
+2. The CNI service deletes the Scaled Endpoint Object.
```

Comment:
Remove these. The CNI is not involved in the scaled endpoint delete. We have a built-ins operator that watches for services/pods. We will need to add a workflow for the builtin operator.

Reply:
Gotcha, Fixed.